packages/frontend/src/features/routes/top-page/type.ts

```diff
@@ -0,0 +1,8 @@
+export type TopPageData = {
+  body: {
+    points: {
+      sendPoint: number;
+      receivedPoint: number;
+    };
+  };
```

Comment:
ご丁寧にありがとうございます！！一旦はここまでラップせずsendPointとreceivedPointのみを持っていればいいです🙇‍♂️
```suggestion
  sendPoint: number;
  receivedPoint: number;
  };
```

Reply:
[4149462](https://github.com/yusei53/monorepo-app/pull/22/commits/4149462c1900d8e8f8d57e2d3d23c4e1ed9b1801)で対応しました！